Makefile

```diff
@@ -0,0 +1,4 @@
+FISTBOOT_SERVICE := $(shell base64 -w0 < aux/custom-first-boot.service)
+
+src/constants.ts: aux/custom-first-boot.service
+	sed -i "s/.*FIRST_BOOT_SERVICE_DATA.*/export const FIRST_BOOT_SERVICE_DATA = '$(FISTBOOT_SERVICE)';/" $@
```

Comment:
It's a multiline string first is the variable and second line the actual string :/
We'll need bit more magic here, time for `awk`? xD

Also, npm is kinda Makefile in js project, but that's secondary, happy to convert it for you once we're happy with it :)

Reply:
Do what you want, tho, curious if I would write a simple bash script would you still recommend to replace it with `npm`? I mean, `make` is older than `bash` sure (like 50 vs 35 years) but what makes people to be so cautious about it. It is a great tool for building and still relevant in 2024. Let me create few targets so `make` calls `npm` and the tools could live peacefully. :-D

But if you want consistency go for it, no hard feelings. Beat my three lines! 😈